src/transformers/configuration_bart.py

```diff
@@ -26,6 +26,7 @@
     "bart-large": "https://s3.amazonaws.com/models.huggingface.co/bert/facebook/bart-large/config.json",
     "bart-large-mnli": "https://s3.amazonaws.com/models.huggingface.co/bert/facebook/bart-large-mnli/config.json",
     "bart-large-cnn": "https://s3.amazonaws.com/models.huggingface.co/bert/facebook/bart-large-cnn/config.json",
+    "bart-tiny-random": "https://s3.amazonaws.com/models.huggingface.co/bert/facebook/bart-tiny-random/config.json",
```

Comment:
Better to upload it under `sshleifer/` (like I have a few test models under `julien-c`), and you don't even need to add references to those files in configuration/modeling.

i.e. just load them with `"sshleifer/bart-tiny-random"`

Reply:
That's way cleaner, thanks!